tests/acceptance/features/apiGraph/userGDPRExport.feature

```diff
@@ -454,3 +454,113 @@ Feature: user GDPR (General Data Protection Regulation) report
       }
     }
     """
+
+
+  Scenario: generate a GDPR report after the admin updates the quota of personal space
+    Given user "Admin" has changed the quota of the personal space of "Alice Hansen" space to "10000"
+    When user "Alice" exports her GDPR report to "/.personal_data_export.json" using the Graph API
+    And user "Alice" downloads the content of GDPR report ".personal_data_export.json"
+    Then the HTTP status code of responses on each endpoint should be "201, 200" respectively
+    And the downloaded JSON content should contain event type "events.SpaceUpdated" in item 'events' and should match
+    """
+    {
+      "type": "object",
+      "required": [
+        "event"
+      ],
+      "properties": {
+        "event" : {
+          "type": "object",
+          "required": [
+            "Executant",
+            "Space"
+          ],
+          "properties": {
+            "Executant": {
+              "type": "object",
+              "required": [
+                "idp",
+                "opaque_id",
+                "type"
+              ],
+              "properties": {
+                "idp": {
+                  "type": "string",
+                  "pattern": "^%base_url%$"
+                },
+                "opaque_id": {
+                  "type": "string",
+                  "pattern": "^%user_id_pattern%$"
+                },
+                "type": {
+                  "type": "number",
+                  "enum": [1]
+                }
+              }
+            },
+            "Space": {
+              "type": "object",
+              "required": [
+                "name",
+                "quota",
+                "space_type"
+              ],
+              "properties": {
+                "name": {
+                  "type": "string",
+                  "enum": ["Alice Hansen"]
+                },
+                "quota": {
+                  "type": "object",
+                  "required": [
+                    "quota_max_bytes",
+                    "quota_max_files"
+                  ],
+                  "properties": {
+                    "quota_max_bytes": {
+                      "type": "number",
+                      "enum": [10000]
+                    },
+                    "quota_max_files": {
+                      "type": "number",
+                      "enum": [18446744073709552000]
+                    }
+                  }
+                },
+                "space_type": {
+                  "type": "string",
+                  "enum": ["personal"]
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    """
+
+
+  Scenario Outline: user tries to generate a GDPR report of other users
```

Comment:
```suggestion
  Scenario Outline: user can't  generate a GDPR report of other users
```
may be we can say like this

Reply:
Wherever possible, let the scenario describe what it is doing or tries to do.
So, use `Scenario: share a file` instead of `Scenario: can share a file`